icons/banknote-arrow-down.json

```
@@ -0,0 +1,19 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "AnnaSasDev",
+    "joffx"
```

Please also add contributors of the base icon (`banknote`):
```suggestion
    "AnnaSasDev",
    "joffx",
    "ericfennis",
    "danielbayley"
```